Solutions/Marain.UserNotifications.Deployment/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Endjin.RecommendedPractices" version="1.2.0" developmentDependency="true" />
+  <package id="Endjin.RecommendedPractices.GitHub" version="1.2.0" developmentDependency="true" />
```

Comment:
This is on an older version than everything else, but actually I don't think any of the packages in this file should be in this project. This is a deployment project that contains no .NET code: it's all ARM templates and PowerShell scripts. (It happens to use the `csproj` file format but that's really only because there isn't a project file format specific to this sort of deployment project.)

So I think the right thing to do is actually to remove everything in here. In fact if VS and the hosted build seem happy to run without this file we should delete it entirely.